### PR TITLE
Add queryByStyle

### PR DESCRIPTION
**Pull request by opsb**

### CODE REVIEW

File: documentation.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
+[
```
Comment (eeue56) [inline documentation.json]: 💭 why did you commit this?

**Review by eeue56 — CHANGES_REQUESTED** [review]
Can you add a test for this?

**Comment by eeue56** [comment]
Oh, actually, no test is needed. But please remove the documentation.json :)

File: documentation.json (hunk 1 above)
Comment (opsb) [inline documentation.json]: Ah missed that one, I've got the elm-format beta installed and had to roll back a bunch of style changes.

Comment (opsb) [reply documentation.json]: I hadn't noticed that it wasn't included, removed now.

**Comment by opsb** [comment]
@eeue56 documentation.json removed

**Comment by eeue56** [comment]
published as 3.0.0

**Pull request merged**